Sources/TemplateKit/Data/TemplateData.swift

```diff
@@ -0,0 +1,193 @@
+import Async
+import Dispatch
+import Foundation
+
+/// Data structure for passing data
+/// into Leaf templates as a context.
+public enum TemplateData {
```

Comment:
Can we potentially break the API by adding more data types?
If so, we should add a comment. If not, let's take a second to see what data types might be missing

Reply:
yes it would be breaking. now is a good time to consider if this enum is complete